src/modules/companies/dto/create-company.dto.ts

```diff
@@ -19,6 +19,7 @@ export class CreateCompanyDto {
   })
   companyName: string;
 
+  @IsOptional()
   @IsNotEmpty()
```

Comment:
optional ou is not empty ?

Reply:
![image](https://github.com/SouJunior/devit-se-backend/assets/83041269/5015557d-6c9a-475d-81ef-6806edc05811)

Por algum motivo, o cnpj não foi exigido como obrigatório.

Reply:
Verifiquei e o campo pode ser considerado como obrigatório, depois a história será alterada.
https://discord.com/channels/1039135265966526525/1082715484597800961/1161629894845943848

Reply:
Maravilha, Igor, muito obrigado.

Reply:
Esqueci de avisar, Wanderson, mas já foi alterado.